### PR TITLE
Shift account menu order

### PR DESCRIPTION
**Pull request by anikbrazeau**
# Summary | Résumé

Move `Your services` above `Your profile`

### CODE REVIEW

**Comment by anikbrazeau** [comment]
[Missing French in Profile section and other small copy changes](https://trello.com/c/D36SQNOd/382-missing-french-in-profile-section-and-other-small-copy-changes)

**Comment by anikbrazeau** [comment]
[Content updates to clarify trail mode](https://trello.com/c/N5fK1VMH/390-content-updates-to-clarify-trail-mode)

File: app/templates/partials/nav/account_menu.html [hunk 1]
```diff
@@ -37,24 +37,24 @@
...
+        aria-label="{{ _('Your services') }}"
+        href="{{ url_for('main.choose_account') }}"
         class="block leading-none text-blue visited:text-blue link:text-blue no-underline text-small"
         role="menuitem">
-        {{ _('Your profile') }}
+        {{ _('Your services') }}
       </a>
     </div>
     <div class="float-right menu-divider"></div>
+    {% endif %}
```
Comment (jimleroyer) [inline app/templates/partials/nav/account_menu.html:50]: Should the _Your services_ menu only be available for the administrator only? If not, we'd need to move back the `{% endif %}` instruction back to line 40.

**Review by jimleroyer — COMMENTED** [review]
Always surprised to see how far you go. I left a comment though on the visibility of the _Your services_ menu, as this could potentially only be seen by administrator in current state.

File: app/templates/partials/nav/account_menu.html (hunk 1 above)
Comment (anikbrazeau) [reply app/templates/partials/nav/account_menu.html]: Merci 👍

**Review by jimleroyer — APPROVED** [review]

**Pull request merged**